SendWithUs.Client/SendWithUsClient.cs

```diff
@@ -43,13 +44,18 @@ public class SendWithUsClient : ISendWithUsClient
         /// <summary>
         /// The base URI of the SendWithUs service.
         /// </summary>
-        /// <remarks>MUST have a trailing slash.</remarks>
-        protected const string BaseUri = "https://api.sendwithus.com/api/v1/";
+        /// <remarks>MUST NOT have a trailing slash.</remarks>
+        protected const string BaseUri = "https://api.sendwithus.com";

```

Comment:
Highly recommend this be changed to `static readonly`, exposed through a read-only property, made `internal`, or made `private`.